### ISSUE TITLE
В версии 1.0.0 не работает отключение изображений

### ISSUE THREAD

**Issue by vistar** [veks/button-visually-impaired-javascript#19]
Происходит только обводка точками, а сами изображения видны.

В предыдущей версии работало это.

Скачано из списка версий.

**Comment by veks** [comment]
Обновил плагин!

**Comment by vistar** [comment]
Да, так лучше, почти...  Теперь изображения вообще выключаются, без обводки, как в ранней версии.
